apps/api/src/services/data-provider/ghostfolio-scraper-api/ghostfolio-scraper-api.service.ts

```diff
@@ -15,6 +15,8 @@ import { PrismaService } from '../../prisma.service';
 
 @Injectable()
 export class GhostfolioScraperApiService implements DataProviderInterface {
+  private static NUMERIC_REGEXP = /[-]{0,1}[\d]*[.]{0,1}[\d]+/g;
```

Comment:
A bit easier and also accepting commas:
`private static NUMERIC_REGEXP = /-?\d*(\.|\,)?\d+/g;`

Reply:
Thanks, I will add support for commas 👍🏻